src/Sylius/Bundle/OrderBundle/DependencyInjection/SyliusOrderExtension.php

```diff
@@ -41,5 +39,10 @@ public function load(array $config, ContainerBuilder $container)
         foreach ($configFiles as $configFile) {
             $loader->load($configFile);
         }
+
+        $orderItemQuantityDataMapper = $container->getDefinition('sylius.data_mapper.order_item_quantity');
+        $orderItemType = $container->getDefinition('sylius.form.type.order_item');
+
+        $orderItemType->addArgument($orderItemQuantityDataMapper);
```

Comment:
It's already fixed in related PR https://github.com/Sylius/Sylius/pull/3842#discussion-diff-49657641.  Shall I fix this here too?